Meow/Constants/APIConstants.cs

```diff
@@ -3,7 +3,7 @@ namespace Meow.Constants;
 public static class APIConstants
 {
     public const string APIBaseUrl = "https://api.thecatapi.com/v1/";
-    public const string APIKey = "THECAT_API_KEY_HERE";
+    public const string APIKey = "live_HzF8UyYCwcjqY05axorP4F5UFp20fewj8xV5aNX8RTpPkd078qsX7umR5Cqy85qw";
```

Comment:
This part I forgot to change back, my bad!

Reply:
Don't worry, I'll make the change in the next commit. I appreciate your contribution.